resources/views/engagements/leave.blade.php

```diff
@@ -19,26 +19,30 @@
                                 'support_person_name' => $engagement->connector->user->support_person_name,
                             ]) }}</strong><br />
                         @if ($engagement->connector->contact_email)
-                            <x-contact-point type='email' :value="$engagement->connector->contact_email" :preferred="$engagement->connector->preferred_contact_method === 'email' &&
-                                $engagement->connector->contact_phone" />
+                            <x-contact-point type='email' :value="$engagement->connector->contact_email"
+                                preferred="{{ $engagement->connector->preferred_contact_method === 'email' && $engagement->connector->contact_phone }}" />
```

Comment:
Are we sure this works? I think I remember there being a distinction where Blade components couldn't use this syntax but can't find documentation on it now.

Reply:
I have tried to test all these types of changes and I believe they are working. For example I was able to get the appended `(preferred)` to flip between email and phone by changing the related setting. It's worth testing out at least one of these changes again though. I think maybe org or fro contact might be easier.